src/openrct2/actions/BannerSetColourAction.hpp

```diff
@@ -103,7 +103,7 @@ DEFINE_GAME_ACTION(BannerSetColourAction, GAME_COMMAND_SET_BANNER_COLOUR, GameAc
 
             auto banner = GetBanner(index);
             banner->colour = _primaryColour;
-            map_invalidate_tile_zoom1(_loc.x, _loc.y, _loc.z, _loc.z + 32);
+            map_invalidate_tile_zoom1({ _loc, _loc.z, _loc.z + 32 });
```

Comment:
This looks odd. In BannerRemoveAction, the z value is divided by 8, but here, it isn't.

Reply:
I think the weirdness is in `BannerRemoveAction.hpp` instead, because this function receives `CoordsXYZZ` and all other places actually seem to send Z coords that are multiplied by 8

Reply:
Looks like it, yeah. Can you correct the call in BannerRemoveAction.hpp, then?

Reply:
Done